.vib/jenkins/cypress/cypress/integration/jenkins_spec.js

```diff
@@ -0,0 +1,94 @@
+/// <reference types="cypress" />
+import { random, checkErrors } from './utils';
+
+it('allows user to log in and log out', () => {
+  cy.login();
+  cy.contains('.alert-danger').should('not.exist');
+  cy.get('div#tasks');
+
+  cy.get('[href$="/logout"]').click();
+  cy.get('form[name*="login"]');
+});
+
+it('should be possible to create a new Git Project', () => {
+  cy.login();
+  cy.visit('/view/All/newJob');
+
+  cy.fixture('items').then((item) => {
+    cy.get('#name').type(`${item.freestyleProject.name}-${random}`);
+    cy.contains(item.freestyleProject.type).click();
+    cy.contains('button', 'OK').should('be.enabled').click();
+    cy.contains('Error').should('not.be.visible');
+
+    cy.get('.radio-block-start').contains('Git').click();
```

Comment:
```suggestion
    cy.contains('.radio-block-start','Git').contains('Git').click();
```

Reply:
It turns out this is NOT equivalent

_Current approach_
<img width="782" alt="Screenshot 2022-05-16 at 12 06 32" src="https://user-images.githubusercontent.com/24253844/168572108-6c445acd-5701-40cb-9d49-fdfa175be2b9.png">

_Your suggestion_
<img width="1184" alt="Screenshot 2022-05-16 at 12 07 18" src="https://user-images.githubusercontent.com/24253844/168572149-f1f7dba7-82e6-4098-9a9a-29f1aab4b4a0.png">

As for the latter, clicking the entire diff does not work. Hence, I will keep the original

Reply:
[According to the Cypress docs](https://docs.cypress.io/api/commands/contains#Syntax), it should be fine to add the selector + text combination in the `contains`.
I don't know if this
`cy.get('.radio-block-start').contains('Git').click();`
will be the same to this
`cy.get('.radio-block-start','Git').click();`
bit I always used them interchangeably. 
However, it's OK from my side to leave it like it is.